app/src/main/java/ch/hikemate/app/ui/saved/SavedHikesScreen.kt

```diff
@@ -172,25 +174,28 @@ private fun PlannedHikes(hikes: List<StateFlow<Hike>>?, hikesViewModel: HikesVie
 @Composable
 private fun SavedHikes(savedHikes: List<StateFlow<Hike>>?, hikesViewModel: HikesViewModel) {
   val context = LocalContext.current
-  Text(
-      context.getString(R.string.saved_hikes_screen_saved_section_title),
-      style = MaterialTheme.typography.titleLarge,
-      modifier = Modifier.padding(16.dp).testTag(SavedHikesScreen.TEST_TAG_SAVED_HIKES_SAVED_TITLE))
-
-  if (savedHikes.isNullOrEmpty()) {
-    Box(modifier = Modifier.fillMaxSize(), contentAlignment = Alignment.Center) {
-      Text(
-          text = context.getString(R.string.saved_hikes_screen_saved_section_empty_message),
-          style = MaterialTheme.typography.bodyLarge,
-          modifier =
-              Modifier.padding(16.dp)
-                  .testTag(SavedHikesScreen.TEST_TAG_SAVED_HIKES_SAVED_EMPTY_MESSAGE))
-    }
-  } else {
-    LazyColumn {
-      items(savedHikes.size, key = { savedHikes[it].value.id }) { index ->
-        val hike by savedHikes[index].collectAsState()
-        SavedHikeCardFor(hike, hikesViewModel)
+  Column(modifier = Modifier.fillMaxSize()) {
```

Comment:
Suggestion: Here too:
```suggestion
// Use fillMaxSize() to ensure consistent layout when switching between tabs
// This prevents unwanted UI resizing that can occur when sections have different
// numbers of items, particularly when transitioning between saved and planned hikes
  Column(modifier = Modifier.fillMaxSize()) {
```

Reply:
Good idea!